lib/agent/https_proxy_ocsp_agent.js

```diff
@@ -98,7 +98,25 @@ function connect(req, opts, fn)
 {
   var proxy = this.proxy;
   var agent = this;
-  Logger.getInstance().debug("Using proxy=%s for host %s", proxy.host, opts.host);
+
+  // try and detect if there's proxy envvar set, currently it's not logged anywhere
+  if (process.env.HTTP_PROXY || process.env.HTTPS_PROXY) {
+    envHttpProxy = process.env.HTTP_PROXY ? 'HTTP_PROXY: ' + process.env.HTTP_PROXY : 'HTTP_PROXY: <unset>'
```

Comment:
we should declare new variables with let/const

Reply:
good point, thanks